dev/com.ibm.ws.logging.osgi/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -91,4 +91,7 @@ console.format.simple=Use the simple logging format.
 console.format.json=Use the JSON logging format.
 
 json.fields=JSON field names that are being replaced
-json.fields.desc=When logs are in JSON format, use this attribute to replace default field names with new field names or to omit fields from the logs. To replace a field name, configure the new field name by using the following format: defaultFieldName:newFieldName?. For field names that are associated with logs of a specified source, use the following format: [source:]?defaultFieldName:newFieldName?, where [source] is the source you want to specify, such as message, trace, or accessLog. To omit a field from the logs, specify the field name without a replacement, as shown in the following example: defaultFieldName:. To rename or omit multiple fields, specify a comma-separated list of field name mappings.
\ No newline at end of file
+json.fields.desc=When logs are in JSON format, use this attribute to replace default field names with new field names or to omit fields from the logs. To replace a field name, configure the new field name by using the following format: defaultFieldName:newFieldName?. For field names that are associated with logs of a specified source, use the following format: [source:]?defaultFieldName:newFieldName?, where [source] is the source you want to specify, such as message, trace, or accessLog. To omit a field from the logs, specify the field name without a replacement, as shown in the following example: defaultFieldName:. To rename or omit multiple fields, specify a comma-separated list of field name mappings.
+
+apps.write.json=Allow apps to write JSON
+apps.write.json=When logs are in JSON format, if the log itself is pre-formatted in JSON, write the logs directly to System.out
```

Comment:
it's not just to System.out... could be System.err or could be messages.log.  will this also apply when someone logs (using JUL) a JSON message?
perhaps something like...
`When message log or console is in JSON format, allow applications to write JSON-formatted messages to those destinations, without modification.`